src/ResourceManager/Sql/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 ## Current Release
 
 ## Version 3.4.1
+* Adding support for list and cancel the asynchronous updateslo operation on the database
```

Comment:
@jiawang003 please move this section to be under `Current Release`

Reply:
fixed